tests/fgnhg_test.py

```diff
@@ -9,187 +9,45 @@
-|                 |                    | 17             |
-|                 |                    | 18             |
-|                 |                    | 19             |
-|                 |                    | 20             |
-|                 |                    | 21             |
-|                 |                    | 22             |
-|                 |                    | 23             |
-|                 |                    | 24             |
-|                 |                    | 25             |
-|                 |                    | 26             |
-|                 |                    | 27             |
-|                 |                    | 28             |
-|                 |                    | 29             |
-|                 |                    | 30             |
-|                 |                    | 31             |
-+-----------------+--------------------+----------------+
-| 100.50.25.12/32 | 200.200.200.4      | 0              |
-|                 |                    | 1              |
-|                 |                    | 2              |
-|                 |                    | 3              |
-|                 |                    | 4              |
-|                 |                    | 5              |
-|                 |                    | 6              |
-|                 |                    | 7              |
-|                 |                    | 8              |
-|                 |                    | 9              |
-|                 |                    | 10             |
-|                 |                    | 11             |
-|                 |                    | 12             |
-|                 |                    | 13             |
-|                 |                    | 14             |
-|                 |                    | 15             |
-+-----------------+--------------------+----------------+
-| 100.50.25.12/32 | 200.200.200.5      | 16             |
-|                 |                    | 17             |
-|                 |                    | 18             |
-|                 |                    | 19             |
-|                 |                    | 20             |
-|                 |                    | 21             |
-|                 |                    | 22             |
-|                 |                    | 23             |
-|                 |                    | 24             |
-|                 |                    | 25             |
-|                 |                    | 26             |
-|                 |                    | 27             |
-|                 |                    | 28             |
-|                 |                    | 29             |
-|                 |                    | 30             |
-|                 |                    | 31             |
-+-----------------+--------------------+----------------+
+FG NHG Prefix    Next Hop            Hash buckets
```

Comment:
We may end up with a lot of hash buckets on one line, making it hard to parse, perhaps we can limit how many hash buckets we print on 1 line, eg at most 30-50chars on one line